Cargo.toml

```diff
@@ -11,7 +11,7 @@ categories = ["concurrency"]
 edition = "2018"
 
 [dependencies]
-parking_lot_core = { path = "core", version = "0.6.2" }
+parking_lot_core = { path = "core", version = "0.7.0" }
 lock_api = { path = "lock_api", version = "0.3.1" }
```

Comment:
I'm not bumping the dependency on `lock_api` here. Because the new `parking_lot` does not *need* the new release. It's just some documentation fixes.